doc/releasenotes.html

```diff
@@ -59,6 +59,55 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
+<li>Fix PV time series shading data import from 3D shade calculator and overlapping inputs in Edit Shading window for PVWatts. (<a href="https://github.com/NREL/SAM/pull/1712">SAM 1712</a>)</li>
+<li>Fix Detailed PV non-linear self shading for vertical arrays to avoid negative POA irradiance values. (<a href="https://github.com/NREL/ssc/pull/1127">SSC 1127</a>)</li>
+<li>Fix issue with Detailed PV subhourly clipping loss when weather file has no DNI data. (<a href="https://github.com/NREL/ssc/pull/1126">SSC 1126</a>)</li>
+<li>Fix reporting of ground-reflected rear irradiance gains in loss diagram for Detailed PV model with bifacial modules. (<a href="https://github.com/NREL/ssc/pull/1125">SSC 1125</a>)</li>
+<li>Add snow slide coefficient to outputs for Detailed PV with snow model enabled. (<a href="https://github.com/NREL/ssc/pull/1128">SSC 1128</a>)</li>
+<li>Clean up PV subhourly clipping loss correction in user interface. (<a href="https://github.com/NREL/SAM/pull/1689">SAM 1689</a>)</li>
+<li>Fix diagrams on Detailed PV Shading and Layout page to show both portrait and landscape instead of duplicating portrait. (<a href="https://github.com/NREL/SAM/pull/1696">SAM 1696</a>)</li>
+</ul>
+</li>
+<li>Concentrating Solar Power (CSP)
+<ul>
+<li>Fix parabolic trough loop configuration on SCA (Collectors) page. (<a href="https://github.com/NREL/SAM/pull/1709">SAM 1709</a>)</li>
+<li>Fix problems with parabolic trough dispatch optimization and loop configuration that caused a simulation error. (<a href="https://github.com/NREL/SAM/issues/1674">SAM 1674</a>, <a href="https://github.com/NREL/ssc/pull/1121">SSC 1121</a>)
+</ul>
+</li>
+<li>Battery Storage
+<ul>
+<li>Improve REopt API calls for battery size and dispatch optimization to better report status and errors from API. (<a href="https://github.com/NREL/SAM/pull/1714">SAM 1714</a>)</li>
+<li>Fix simulation error caused by battery environment temperature input as time series array. (<a href="https://github.com/NREL/SAM/pull/1694">SAM 1694</a>, <a href="https://github.com/NREL/ssc/pull/1138">SSC 1138</a>)</li>
+<li>Fix issue with behind-the-meter battery with retail rate dispatch that caused a simulation error. (<a href="https://github.com/NREL/ssc/pull/1135">SSC 1135</a>)
+<li>Fix default battery life inputs so correct table is shown for the default Li-ion LFP/Graphite battery type.  (<a href="https://github.com/NREL/SAM/pull/1715">SAM 1715</a>)
+<li>Fix problem with battery discharging to grid for manual dispatch option. (<a href="https://github.com/NREL/ssc/pull/1115">SSC 1115</a>)</li>
+<li>Fix problem with battery self consumption dispatch option with grid outage. (<a href="https://github.com/NREL/ssc/pull/1114">SSC 1114</a>)</li>
+</ul>
+</li>
+<li>Fix problem with losses specifed as "time series losses with custom time periods" that caused a simulation error. (<a href="https://github.com/NREL/SAM/pull/1698">SAM 1698</a>)</li>
```

Comment:
It looks like "financial models header" might be missing? (not sure, reading this in html)

https://github.com/NREL/ssc/pull/1129 should have been tagged with the patch as well.